### PR TITLE
Quantum fix

### LINKED ISSUE #1217: Quantum doesn't work: Globals only loading last variable in globals config
Using @next, found a weird bug in quantum that prevents my builds from working properly.

My globals config looks like this:

screen shot 2018-04-24 at 12 04 04 pm

I'm using bakeApiIntoBundle too just FYI.

The end of my app.js looks like this:

screen shot 2018-04-24 at 12 04 46 pm

If I change the order of the globals config, then the last one will be the only one that is exposed.

Will investigate further, but this is definitely a quantum related bug.

### PR DESCRIPTION
**Pull request by jpike88**
Allows a globals configuration object to function identically across quantum and non-quantum builds.

Fixes #1217

### CODE REVIEW

File: src/quantum/plugin/FlatFileGenerator.ts [hunk 1]
```diff
@@ -68,25 +68,23 @@ export class FlatFileGenerator {
...
-            const req = `${this.core.opts.quantumVariableName}.r(${JSON.stringify(this.entryId)})`;
 
-            if (this.globalsName) {
-                if (this.core.opts.isTargetNpm() || this.core.opts.isTargetServer()) {
-                    this.contents.push(`module.exports = ${req}`);
-                }
+        this.globals.forEach((fileID, globalName) => {
+            const req = `${this.core.opts.quantumVariableName}.r(${JSON.stringify(fileID)})`;
+            if (this.core.opts.isTargetNpm() || this.core.opts.isTargetServer()) {
+                this.contents.push(`module.exports = ${req}`);
```
Comment (nchanged) [inline src/quantum/plugin/FlatFileGenerator.ts]: It's not gonna work for module.exports here
We need to have only one module exports for server

Comment (nchanged) [reply src/quantum/plugin/FlatFileGenerator.ts]: Also please check the tests

Comment (nchanged) [reply src/quantum/plugin/FlatFileGenerator.ts]: They all need to pass, before we write new ones

Comment (jpike88) [reply src/quantum/plugin/FlatFileGenerator.ts]: Fixed, also brought back in use of "*"

**Comment by nchanged** [comment]
Awesome! I will check it manually today!

**Pull request merged**